capture-sdk/sdk/src/main/res/values-sw600dp/dimens.xml

```diff
@@ -48,5 +48,6 @@
     <dimen name="gc_photo_tip_text_margin">2dp</dimen>
     <dimen name="gc_tablet_width">582dp</dimen>
     <dimen name="gc_snackbar_text_height">60dp</dimen>
+    <dimen name="gc_camera_pan_width">124dp</dimen>
```

Comment:
[Request] Typo: it says pan instead of pane. 🙂

Reply:
Done